db_scripts/dump.sh

```diff
@@ -1,6 +1,17 @@
 #!/bin/bash
 
-# For AMI: sudo yum -y install https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6-x86_64/pgdg-ami201503-96-9.6-2.noarch.rpm
-#          sudo yum -y install postgresql96
+# export ENV_NAME=prod
+# aws cloudformation create-stack --region us-east-1 --stack-name $ENV_NAME-bastionhosts --template-url https://s3.amazonaws.com/crowd-deployment/infrastructure/bastion-hosts.yaml --parameters ParameterKey=EnvironmentName,ParameterValue=$ENV_NAME ParameterKey=KeyPairName,ParameterValue=rstorey@loc.gov --disable-rollback
+# aws cloudformation delete-stack --region us-east-1 --stack-name $ENV_NAME-bastionhosts
 
-pg_dump -Fc --clean --create --no-owner --no-acl -U concordia -h $POSTGRESQL_HOST concordia -f concordia.dmp
\ No newline at end of file
+export TODAY=20190129
+export POSTGRESQL_PW=${POSTGRESQL_PW:-password}
+export POSTGRESQL_HOST=${POSTGRESQL_HOST:-localhost}
+export DUMP_FILE=concordia.dmp
+
+echo "$POSTGRESQL_HOST:5432:*:concordia:$POSTGRESQL_PW" >> ~/.pgpass
```

Comment:
Using `>>` will make this file grow each time it runs. Is this run anywhere other than the dedicated bastion host where we would need to care about having multiple accounts? Perhaps we could simply run this only when `~/.pgpass` does not exist?

Reply:
So the way I've been doing this is to delete the stack after I'm done using it, so it only exists for a few minutes while I'm running the command (maybe I can even change this to a lambda function). I changed this to use `>` so it'll overwrite instead of append, just in case there's a situation where I have to rerun the script on the same host.

Reply:
Yeah, it's not a huge deal but I figured either clobbering it or doing a `if [ ! -f … ]` check avoids some potential confusion later.